sdks/python/apache_beam/runners/portability/fn_api_runner/translations.py

```diff
@@ -75,21 +75,27 @@
 
 IMPULSE_BUFFER = b'impulse'
 
+# SideInputId is identified by a consumer ParDo + tag.
+SideInputId = Tuple[str, str]
+
+DataSideInput = Dict[SideInputId,
```

Comment:
What does the value represent?

Reply:
The value is a tuple with the encoded data. I've moved these to translations.py, and updated the comment